src/report/report.py

```diff
@@ -571,6 +571,7 @@ def _get_issue_labels(
         job_name: Optional[str],
         type: str,
         jira_additional_labels: Optional[list[str]],
+        jira_additional_labels_filepath: str = f"{os.getenv('SHARED_DIR', '')}/firewatch-additional-labels",
```

Comment:
I have one more approach to suggest, I'm not sure if it's better but worth mention:

Shouldn't we set the filepath as a cli input with click and then handle setting it's value with the `$SHARED_DIR/firewatch-additional-labels` file within the openshift-ci end?

I.e. Openshift-Ci will have a ref that handles extracting all the labels in bash into a file, then set the `jira_additional_labels_filepath` value in the `firewatch-report-issues` script

Reply:
Another thought, can't we handle it completly on Openshift-Ci end by modifying the lables env var and extending it in a dedicated ref?

I mean user will set the config file with:
```
FIREWATCH_DEFAULT_JIRA_ADDITIONAL_LABELS: '["4.15-lp","rosa-classic-lp"]'
```
And then one more ref will parse and update the env var in bash to be set with:
```      
FIREWATCH_DEFAULT_JIRA_ADDITIONAL_LABELS: '["4.15-lp","rosa-classic-lp", "added-label","one-more-label"...]'
```

Reply:
I like the idea of setting it in the CLI and passing the file in OpenShift CI. I think we should avoide adding another step to parse the file, though I think it would work. I will make changes to pass the file in the CLI :)